TOC.md

```diff
@@ -236,6 +240,7 @@
     - [TiKV 调优](dev/reference/performance/tune-tikv.md)
     - [TiDB 最佳实践](https://pingcap.com/blog-cn/tidb-best-practice/)
   + [TiSpark 使用指南](dev/reference/tispark.md)
+  + [TiDB Binlog 简介](dev/reference/tidb-binlog-overview.md)
```

Comment:
This renaming affects links in the following files. Please update them to the new link to avoid possible issues:

- `dev/how-to/deploy/orchestrated/ansible.md`: `/dev/reference/tools/tidb-binlog/overview.md`
- `dev/how-to/get-started/tidb-binlog`: `/dev/reference/tools/tidb-binlog/overview.md`
- `dev/reference/tools/download.md`: `/dev/reference/tools/tidb-binlog/overview.md`, /dev/reference/tools/tidb-binlog/overview.md#binlogctl-guide